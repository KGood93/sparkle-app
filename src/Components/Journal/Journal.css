.disription {
    border: solid rgb(7,123,136) 3px;
    background-color: rgb(241,221,210);
    width: 250px;
    margin: 10px;;
}

.descripText {
    font-family: 'Playfair Display', serif;
    margin: 10px;
}