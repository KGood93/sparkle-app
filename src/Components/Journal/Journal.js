import React from 'react';
//import AddButton from '../AddButton/AddButton.js'
import './Journal.css';

class Journal extends React.Component {

    render() {
        return (
          <div className="main"> 
          <div class="entryButton">
            <button type="button" class="addEntry">Add Entry</button>
          </div>
            <section className="entries">
                <div className="entryOdd">
                    <h3 className="entryName">Entry Title 1</h3>
                    <h4 className="entryDate">Date</h4>
                    <button type="button" className="Open">Open</button>
                    <button type="button" className="Delete">Delete</button>
                </div>
                <div className="entryEven">
                    <h3 className="entryName">Entry Title 2</h3>
                    <h4 className="entryDate">Date</h4>
                    <button type="button" className="Open">Open</button>
                    <button type="button" className="Delete">Delete</button>
                </div>
                <div className="entryOdd">
                    <h3 className="entryName">Entry Title 3</h3>
                    <h4 className="entryDate">Date</h4>
                    <button type="button" className="Open">Open</button>
                    <button type="button" className="Delete">Delete</button>
                </div>
                <div className="entryEven">
                    <h3 className="entryName">Entry Title 4</h3>
                    <h4 className="entryDate">Date</h4>
                    <button type="button" className="Open">Open</button>
                    <button type="button" className="Delete">Delete</button>
                </div>
            </section>
          </div>
        )
    }
}
export default Journal;