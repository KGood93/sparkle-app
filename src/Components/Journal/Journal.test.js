import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Journal from './Journal';


describe('Journal Component', () => {
    it("renders the complete journal", () => {
        const wrapper = shallow(<Journal />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
});