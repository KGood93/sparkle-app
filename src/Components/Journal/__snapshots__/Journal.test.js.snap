// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Journal Component renders the complete journal 1`] = `
<div
  className="main"
>
  <div
    className="discription"
  >
    <p
      className="descripText"
    >
      Sparkle is a motivational journaling app to help inspire you to lead your best life. Add a new entry or look back on previous entries to help spark your dreams and creativity.
    </p>
  </div>
  <div
    className="entryButton"
  >
    <AddButton
      className="addEntryButton"
      tag={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/addEntry"
      type="button"
    >
      Add Entry
    </AddButton>
  </div>
  <section
    className="entries"
  >
    <JournalEntry />
  </section>
</div>
`;
